tests_integration/__tests__/__snapshots__/multiple-patterns.js.snap

```diff
@@ -0,0 +1,44 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`multiple patterns 1`] = `
+"directory/file.js
+directory/nested-directory/nested-directory-file.js
+other-directory/file.js
+other-directory/nested-directory/nested-directory-file.js
+"
+`;
+
+exports[`multiple patterns with ignore nested directories pattern 1`] = `
+"directory/file.js
+other-directory/file.js
+other-regular-modules.js
+regular-module.js
+"
+`;
+
+exports[`multiple patterns with ignore pattern, doesn't ignore node_modules with --with-node-modules flag 1`] = `
+"node_modules/node-module.js
+other-directory/file.js
+other-directory/nested-directory/nested-directory-file.js
+other-regular-modules.js
+regular-module.js
+"
+`;
+
+exports[`multiple patterns with ignore pattern, ignores node_modules by default 1`] = `
+"other-directory/file.js
+other-directory/nested-directory/nested-directory-file.js
+other-regular-modules.js
+regular-module.js
+"
+`;
+
+exports[`multiple patterns with ignore pattern, ignores node_modules by with ./**/*.js 1`] = `
+"./directory/file.js
+./directory/nested-directory/nested-directory-file.js
```

Comment:
This tests uses `./**/*.js !directory/**`, right? Then shouldn’t all files in `./directory/` have been excluded?